repositories/activities.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../models')
+
+const create = async (data) => {
+  const createdData = await db.Activities.create(data)
+  return {
```

Comment:
Acá sólo hacer return de lo que devuelve el create, el formato de respuesta corresponde al controller

Reply:
Corregido.
Thank you so much!